appcelerator/titanium_mobile
# [TIMOB-26497] Android: Fix annotation validation

- Fix aggressive Kroll annotation validation
- Amend native module template

###### TEST CASE
- Create Titanium native module project
- Build module: `appc run -p android --build-only`
- Should not see validation errors

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-26497)
